rules/kubecf/release_chart.sh

```diff
@@ -0,0 +1,21 @@
+#!/bin/bash
+
+# This is a helper script which combines a helm chart tarball and an image list
+# into a new tarball.
+
+if [ ! -e "$1" ]; then
+  echo "Helm chart tarball '$1' does not exist, bailing out!"; exit 1
```

Comment:
```suggestion
  >&2 echo "Helm chart tarball '$1' does not exist, bailing out!"; exit 1
```

Reply:
Done in https://github.com/cloudfoundry-incubator/kubecf/pull/634/commits/42b322103f8bd3dc45962383f317702e9d09d476.